test/common.bash

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+# bats setup function
+setup() {
+  export KUBECONFIG=$(mktemp)
```

Comment:
I don't understand why you don't do this like

    KUBECONFIG="$XDG_CACHE_HOME/kubeconfig"

you're gonna get a random dir every time anyway.